### PR TITLE
[MIG] product_supplierinfo_for_customer_sale: Migration to 16.0

### PR DESCRIPTION
**Pull request by Reyes4711-S73**
Standard migration to 16.0

depends on https://github.com/OCA/product-attribute/pull/1230

### CODE REVIEW

**Comment by desdelinux** [comment]
Do you have any progress?

@Reyes4711-S73

**Comment by Reyes4711-S73** [comment]
@desdelinux The test is failing due to the name_search in PR https://github.com/OCA/product-attribute/pull/1230. He needs to restore the name_search function

File: product_supplierinfo_for_customer_sale/tests/test_product_name_search.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def test_10_find_product_customer_code(self):
         self.assertFalse(self.product_1.customer_ids)
 
         # Search by product customer code
-        product_names = self.product.name_search(name="code_test")
+        product_names = self.product.name_search(name="code_test", operator="ilike")
```
Comment (luisg123v) [inline product_supplierinfo_for_customer_sale/tests/test_product_name_search.py]: I think this change is not required, `ilike` is [still the default operator](https://github.com/odoo/odoo/blob/e2f9c21f06837cb7bfdf2547a05633e563164d09/odoo/models.py#L1581) in 16.0

Same for the below similar cases

**Review by luisg123v — CHANGES_REQUESTED** [review]
- there are commits present in 15.0 that are missing here, e.g. 5c02f9d9b20216e09103aa4d05ca691347080673
- Remove `[16.0]` from commit title (in PR title is OK), because it doesn't comply with guidelines.

**Comment by Reyes4711-S73** [comment]
@luisg123v Done. Could you merge it?

File: product_supplierinfo_for_customer_sale/__manifest__.py [hunk 2]
```diff
@@ -4,14 +4,18 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Product Supplierinfo for Customer Sale",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
```
Comment (luisg123v) [inline product_supplierinfo_for_customer_sale/__manifest__.py:7]: Bump version on the migration commit,. In the historic, version was bumped to 15.0.1.1.0 (done on 36bc0da701080b4d8902a2cb98f7118a823d77c2).

You squashed the actual commit and the bumping one correctly, but the version was incorrectly set.

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by Reyes4711-S73** [comment]
@luisg123v Done. Could you merge it?

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

@moylop260 Could you review, please?

**Pull request merged**